Tasks/DockerV2/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -30,20 +29,22 @@
   "loc.messages.DeletingDockerConfigDirectory": "Deleting Docker config directory. Path: %s",
   "loc.messages.DeletingAuthDataFromDockerConfig": "Deleting auth data for registry from Docker config file. Registry: %s, New Docker config: %s",
   "loc.messages.DockerRegistryNotFound": "Docker registry service connection not specified.",
+  "loc.messages.ErrorParsingDockerConfig": "Could not parse the Docker config obtained from the file. Error: %s",
   "loc.messages.FileContentSynced": "Synced the file content to the disk. The content is %s.",
   "loc.messages.FoundDockerConfigStoredInTempPath": "Found the Docker Config stored in the temp path. Docker config path: %s, Docker config: %s",
   "loc.messages.FoundLoginsForOtherRegistries": "Found login info for other registry(s). Trying to remove auth from the Docker config for the registry: %s",
-  "loc.messages.OldDockerConfigContent": "Found an earlier login to the same registry. Saving old auth data and continuing with the new login. Old docker config: %s",
+  "loc.messages.IgnoringArgumentsInput": "The arguments input is not supported when the command is buildAndPush. Ignoring the input.",
```

Comment:
where is this message used?

Reply:
It is used in dockerbuildandpush.ts, which I somehow missed pushing :/ Pushed now.